src/recipients/recipient.ts

```diff
@@ -252,7 +252,7 @@ function executeFuzzySearch(
   shouldSort?: boolean
 ): FuzzyRecipient[] {
   const parsedQuery = query.replace(/[()-\s/\\]/g, '')
-  if (parsedQuery === '' || parsedQuery.length < 2) {
+  if (parsedQuery === '') {
```

Comment:
this looks like it could be valuable, but isn't related to the main purpose of the PR. can you split it out to keep the PR single-purpose?

Reply:
Should be in #4490.